scripts/crontab

```diff
@@ -67,6 +67,7 @@ PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/us
 #0 0 1 */3 * cd gitcoin/coin; bash scripts/run_management_command_if_not_already_running.bash send_quarterly_stats --live  >> /var/log/gitcoin/send_quarterly_stats.log  2>&1
 00 10 * * 5 cd gitcoin/coin; bash scripts/run_management_command_if_not_already_running.bash post_leaderboard_to_slack  >> /var/log/gitcoin/post_leaderboard_to_slack.log  2>&1
 0 9 * * * cd gitcoin/coin; bash scripts/run_management_command_if_not_already_running.bash no_applicants_email  >> /var/log/gitcoin/no_applicants_email.log 2>&1
+0 10 15 1 *  cd gitcoin/coin; bash scripts/run_management_command_if_not_already_running.bash send_tax_report >> /var/log/gitcoin/send_tax_report.log 2>&1
```

Comment:
id prefer to just send this manually instead of via cron

Reply:
Ok no problem, I'm going to delete it.

Reply:
Done